cypress/e2e/admin/adminRegistersTeacherWithLink.feature

```diff
@@ -1,37 +1,69 @@
 @regression_test
-@unstable_test
+@stable_test
 Feature: Teacher registration with registration link send by admin
 
     As a teacher, I want to register in the SVS
 
-    Scenario Outline: Setup
-        Given I am logged in as '<user>' on '<env>'
-        When Created teacher 'John' 'Teacher' with email 'john.teacher@mail.tld'
-        Then I am on the teachers management page
+    Scenario: Admin add a new teacher and register it via registration link
 
-        Examples:
-            | env | user       |
-            | dbc | admin1_dbc |
+        # pre-condition: admin logs in to create their account in a school
+        Given I am logged in as a '<admin>' at '<namespace>'
 
-    Scenario Outline: Teacher can registers in the SVS with the registration link
-        Given I am logged in as '<user>' on '<env>'
-        And I am on the teachers management page
-        When Going to teacher edit page for 'John' with email 'john.teacher@mail.tld'
-        And Going to teacher registration page
-        And I am registering as teacher with email 'john.teacher@mail.tld' and password 'Password1!' on '<env>'
-        Then I am able to logout
+        # Admin creates a new teacher
+        When I click on administration in menu
+        When I go to '<role_to_manage>' administration
+        When I click on FAB
+        When I click on Add User in opened FAB for '<role_to_manage>'
+        When I fill out the user creation form for '<user_firstname>' '<user_lastname>' with email '<user_email>'
+        When I click on add button to add '<role_to_manage>'
+        When I enter '<role_to_manage>' email '<user_email>' in search input field
+        Then I can see the user with email '<user_email>' in the table
 
-        Examples:
-            | env | user       |
-            | dbc | admin1_dbc |
+        # Admin generates the registeration link to proceed with teacher registration
+        When I enter '<role_to_manage>' email '<user_email>' in search input field
+        When I click edit '<role_to_manage>' button for '<user_email>'
+        Then I click on the button Generate Personal Registration Link for teacher
+
+        # Teacher performs registration via the generated link with PIN
+        Then I visit to the generated student registration link
```

Comment:
wrong step here we are in the process of registering a teacher and not student.

Reply:
done!